### DISCUSSION TITLE
Custom GPG only for Debian Family

### DISCUSSION THREAD

**Discussion by gabardyne** [uyuni-project/uyuni#6178] [Q&A]
Hello,

Is there a way to use official gpg keys on channels for Debian Family?

I fill the "GPG key URL" field with this for example : file:///etc/apt/trusted.gpg.d/debian-archive-bullseye-automatic.gpg.

But nothing happen, even if I sync the repo, and reapply highstate on the client.

Am I missing something?

Gab

**Comment by mcalmer** [comment]
This is not really possible with Uyuni.
Debian sign only the metadata and Uyuni re-generate the metadata. So the signature would not match.
You can create your own GPG key in Uyuni and re-sign the metadata generated from Uyuni. 
This GPG key is automatically deployed on the clients and configured to be checked.

Look into the Uyuni documentation and search for "signing metadata"
